tools/istio-iptables/pkg/cmd/root_test.go

```diff
@@ -84,14 +97,32 @@ func TestGetLocalIP(t *testing.T) {
 					netip.MustParseAddr("2222:3333::1"),
 				})
 			},
-			isDS:     true,
+			enableDS: viaEnv,
+			expected: true,
+		},
+		{
+			name: "mixed ipv4 and ipv6 local ip addresses with dual stack enable via redirect",
+			lipas: func() ([]net.Addr, error) {
+				return tesrLocalIPAddrs([]netip.Addr{
+					netip.MustParseAddr("::1"),
+					netip.MustParseAddr("127.0.0.1"),
+					netip.MustParseAddr("1.2.3.5"),
+					netip.MustParseAddr("2222:3333::1"),
+				})
+			},
+			enableDS: viaRedirect,
 			expected: true,
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			LocalIPAddrs = tt.lipas
-			DualStack = tt.isDS
+			switch tt.enableDS {
```

Comment:
use `test.SetForTest`, you can see a lot of examples of overriding fields that were env vars